extensions/search/schema.json

```diff
@@ -0,0 +1,45 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "schema.json#",
+  "title": "Search Extension",
+  "type": "object",
+  "description": "STAC Search Extension",
+  "additionalProperties": true,
+  "allOf": [
+    {
+      "$ref": "#/definitions/search_meta"
+    }
+  ],
+  "definitions": {
+    "search_meta": {
+      "properties": {
+        "properties": {
+          "type": "object",
+          "required": [
+            "next",
+            "returned"
+          ],
+          "properties": {
+            "next": {
+              "type": ["string", "null"]
+            },
+            "limit": {
+              "type": ["number", "null"]
+            },
+            "found": {
+              "type": "number"
+            },
+            "returned": {
+              "type": "number"
+            },
+            "properties": {
```

Comment:
It feels like there are too many `properties` properties nested here. Maybe rename it to `additional` or so?

Reply:
i just removed it all together, since you throw whatever you want into the object anyway.